Marlin/src/module/temperature.cpp

```diff
@@ -531,7 +531,11 @@ int Temperature::getHeaterPower(int heater) {
 
     uint8_t fanDone = 0;
     for (uint8_t f = 0; f < COUNT(fanPin); f++) {
-      int8_t pin = pgm_read_byte(&fanPin[f]);
+      #ifdef ARDUINO
+        int8_t pin = pgm_read_byte(&fanPin[f]);
```

Comment:
Since `pin_t` is `int8_t` on AVR, wouldn't it make sense to go ahead and use `pin_t` here?

Reply:
Will do.